torch/_inductor/ir.py

```diff
@@ -7005,11 +7005,15 @@ def create(cls, x: IRNode, device: torch.device, non_blocking: bool) -> IRNode:
 
         developer_warning("DeviceCopy in input program")
         constant_args = (non_blocking,)
+        # Device Copy should keep the same layout as input
```

Comment:
> Device Copy should keep the same layout as input

nit - could you add something that the aten cuda copy adds an intermediary if both inputs are not contiguous ? (even though it should just require them to have the same memory layout..)

Reply:
I think it is just contiguous.. but let know if you see otherwise somewhere